.github/workflows/additional_tests.yml

```diff
@@ -2,7 +2,7 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-name: Test pip install -e .
```

Comment:
I think the original name is reasonable? The additional things are tested in lint.

Reply:
Ok